lib/screens/about_page.dart

```diff
@@ -72,9 +74,25 @@ class AboutScreen extends StatelessWidget {
               "Entwickelt von susch19 (Sascha Hering)",
             ),
           ),
-          ListTile(
-            title: Text("Version 0.1"),
-          ),
+          FutureBuilder<VersionAndUrl?>(
+              future: UpdateManager.getVersionAndUrl(),
+              builder: (context, AsyncSnapshot<VersionAndUrl?> snapshot) {
+                return ListTile(
+                    title: Row(children: [
+                      Text(UpdateManager.getVersionString(snapshot.data?.version)),
+                      if (!snapshot.hasData && !snapshot.hasError)
+                        Padding(
+                          padding: const EdgeInsets.only(left: 8),
+                          child: CircularProgressIndicator(),
+                        )
+                    ]),
+                    onTap: () {
+                      if (snapshot.data?.url != null) {
+                        HelperMethods.openUrl(snapshot.data!.url!);

```

Comment:
```suggestion
                        HelperMethods.openUrl(snapshot.data!.url);
```

Same here, should not be nullable in the future